packages/shared/src/components/modals/ClickbaitShieldModal.tsx

```diff
@@ -82,7 +82,7 @@ const ClickbaitShieldModal = ({
           tag="a"
           type="button"
           variant={ButtonVariant.Primary}
-          href={`${webappUrl}plus`}
+          href={hasUsedFreeTrial ? `${webappUrl}plus` : undefined}
```

Comment:
Can't you prevent default on the onClick?

Reply:
Yup 🙈
Changed